source/use_cases/aws-restaurant-management-demo/README.md

```diff
@@ -2,33 +2,55 @@
 
-### Managers
-- List all orders (all statuses)
+#### Managers
+- View all orders (all statuses)
 - Close-out service
   - Generate sales report
   - Calculate tips for servers
-  - Archive orders to the data warehouse
+  - Archive orders to the data warehouse (and clear out the table)
 - Automatic delayed order detection
-- View report(s)
+- View a specific sales report
+
+## CDK project structure
+This CDK project will deploy a total of five (5) stacks into your AWS account using CloudFormation:
+- `ExistingResources` - provisions an Amazon S3 bucket into your account that simulates/demonstrates using the project with existing infrastructure.
+- `SharedStack` - provisions resources that are used across the following three stacks, including a central DynamoDB table (with configuration) and a Lambda layer for common database access functions (i.e. using a table scan to get all orders, archive the table, etc.).
+- `ServiceStaffStack` - provisions resources that are used by service staff to access the system. Provides functions that allow users to create a new order and close out an order.
+- `KitchenStaffStack` - provisions resources that are used by kitchen staff to access the system. Provides functions that allow users to view all open orders and mark a specific order as filled by the kitchen.
+- `ManagerStack` - provisions resources that are used by one or more managers to access the system. Provides functions that allow users to view all orders (regardless of status), close-out the service for a day, and retrieve a specific report saved from a previous day's close-out.
+
+## Setting up
+After cloning this project into your local environment, we recommend performing the following steps to deploy it into 
+your account:
+- Run `npm install` at the root level of the project, to install all dependencies.
+- Run `npm run build` at the root level of the project, to build the project.
+- Run `cdk deploy --all` at the root level of the project, to initiate the deployment.
+
+Note: If you are working on a clean instance or development environment, you will need to perform initial configuration
+steps for the CDK in order to deploy the project. Click [here](https://docs.aws.amazon.com/cdk/latest/guide/getting_started.html) for 
+more information on getting started with the CDK.
 
 ## Business logic
 This demo features multiple Lambda functions, one assigned to each function for each user role. The business logic that
 each of these functions runs can be found organized under the `lib/lambda` folder. Each function is decoupled and managed
-within its own folder, with its own package.json file and dependencies.
+within its own folder.
 
 ## API
-This demo implements three REST APIs using AWS Gateway, one for each user type. Mappings are provided as follows:
+This demo implements three REST APIs using AWS Gateway, one for each user type. For sample requests of each type, please
+refer to the `SAMPLE_REQUESTS.md` file. Mappings are provided as follows:
```

Comment:
Does sample requests file also explain how to authenticate in Cognito in order to make the call?